app/src/main/java/ru/sonya/week3/viewModel/MainViewModel.kt

```diff
@@ -27,10 +37,15 @@ class MainViewModel(
     }
 
     private fun getCats() {
-        screenState.value = MainState(repository.getCats())
+        viewModelScope.launch(Dispatchers.IO) {
+            val cats = repository.getCats(_context)
+            val catsList: List<FunCat> =
+                cats!!.map { FunCat(it.url, it.breeds[0].name, it.breeds[0].description) }
```

Comment:
Вот самый простой адекватный вариант, лучше конечно сделать обработку ошибок из-за которых лист null пришёл
```suggestion
            val catsList: List<FunCat> =
                cats?.map { FunCat(it.url, it.breeds[0].name, it.breeds[0].description) }.orEmpty()
```